### DISCUSSION TITLE
Formatting to use in excel to display PitOutTime etc. as readable times.

### DISCUSSION THREAD

**Discussion by laurence9899** [theOehrly/Fast-F1#448] [Q&A] [answered]
Hi all,

I've downloaded some data from the fastf1 library and want to view it in excel in a readable format. I have used the format specifier mm:ss.000 for the laptime data and it looks good but I can't figure out what numerical format specifier to use for the PitOutTime and PitInTimes. I've tried several but all appear to display incorrect times. Any suggestions appreciated.

Thank you!!

**Comment by Casper-Guo** [comment, marked as answer]
![image](https://github.com/theOehrly/Fast-F1/assets/89810860/82d35e3e-9700-4e16-bbb8-800c269157ac)

I have it just as General so maybe no specific format is required? Caveat is this is a csv file opened in Excel instead of a xlsx file.
